### DISCUSSION TITLE
Limiting Sentence Length, Timestamp Duration, and Word Count in Transcriptions

### DISCUSSION THREAD

**Discussion by vengr2525** [chidiwilliams/buzz#954] [Q&A] [answered]
Hello,

First of all, I would like to express my gratitude to your team for creating such a wonderful program! Buzz has been incredibly helpful in my work, and I really appreciate the effort and dedication that has gone into developing it.

I am using Buzz "Whisper (Small)". for transcribing audio files, and I have encountered a few challenges. The resulting sentences tend to be too long, and when used in video editing, the text takes up too much space on the screen.

I would like to configure the output so that:

Each line contains no more than six words.
Timestamps are limited to 3–4 seconds _(e.g., 00:00:23,400 --> 00:00:27,400)_.
Is there a way to set this format using custom prompts or other parameters within the program? If not, could you please advise on how best to achieve this?

Thank you again for your excellent work and for your assistance with this request!

**Comment by raivisdejus** [comment, marked as answer]
@vengr2525 thanks for your kind words. Please see the resize button on top of the transcription viewer screen. The screen where you see the transcriptions.

Currently the resize option is quite basic. More advanced options may come in time. I plan to integrate the `stable-ts` tool to post process whisper results, but it will take some time to materialize.

**Reply by vengr2525** [reply]
> Please see the resize button

🤦‍♂️
🤦‍♂️🤦‍♂️
🤦‍♂️🤦‍♂️🤦‍♂️

I didn't even think about it. Thanks a lot. Now everything is as I wanted. Good luck to you.
